src/main/java/com/zufar/onlinestore/cart/api/CartApiImpl.java

```diff
@@ -43,8 +43,9 @@ public ShoppingSessionDto addNewItemToShoppingSession(final AddNewItemToShopping
     }
 
     @Override
-    public ShoppingSessionDto removeItemFromShoppingSession(final RemoveItemFromShoppingSessionRequest removeItemFromShoppingSessionRequest) {
-        return null;
+    public ShoppingSessionDto removeItemFromShoppingSession(final RemoveItemFromShoppingSessionRequest request) throws
+            ShoppingSessionItemNotFoundException, ShoppingSessionNotFoundException {
```

Comment:
I don't think that it's a good practice to put 2 checked exceptions in the method declaration. If your service is throwing them up please handle it , provide user friendly message and process